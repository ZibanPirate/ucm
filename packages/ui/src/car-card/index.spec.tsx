import { render } from "@testing-library/react";

import { CarCard } from ".";

describe(`Testing component '${CarCard.name}' :`, () => {
  it(`should render with provided linkWrapper as Fragment`, () => {
    const { container } = render(
      <CarCard
        LinkWrapper={() => <></>}
        description={"test description"}
        image="/assets/test.svg"
        make="BMW"
        model="M2"
        price={39000}
      />,
    );
    expect(container).toMatchSnapshot();
  });
});
