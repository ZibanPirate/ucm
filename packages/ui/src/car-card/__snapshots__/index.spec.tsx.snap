// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component 'CarCard' : should render with provided linkWrapper as Fragment 1`] = `
<div>
  <div
    class="ucm-ui-car-card"
    style="background-color: whitesmoke; display: flex; flex-direction: column;"
  >
    <img
      src="/assets/test.svg"
      style="object-fit: cover; width: 100%;"
    />
    <span
      style="font-size: large; margin: 0.3rem .6rem; display: flex;"
    >
      BMW M2
    </span>
    <span
      style="font-size: medium; margin: 0.3rem .6rem; display: flex;"
    >
      â‚¬39000.00
    </span>
    <span
      style="font-size: small; margin: 0.3rem .6rem .6rem; display: flex; flex: 1;"
    >
      test description
    </span>
  </div>
</div>
`;
